2.8.0/done/result_profiler.md

```diff
@@ -23,32 +23,36 @@ The categories below are as follows:
 * Developers: All commits that are not end-user facing but still impact people that compile from source, develop into pytorch, extend pytorch, etc
 * not user facing: All commits that are not public end-user facing and hence should be dropped from the release notes
 
-## profiler
+## Profiler
 ### bc breaking
 ### deprecation
 ### new features
+- [Memory Snapshot] Add Flag to Toggle Global and Local Callbacks for Annotations ([#154932](https://github.com/pytorch/pytorch/pull/154932))
```

Comment:
formatting nit: please remove tags e.g. `[Memory Snapshot]`, `[Profiler]` throughout as suggested in the release notes formatting guide